src/main/scala/org/hammerlab/guacamole/loci/LociSet.scala

```diff
@@ -92,7 +93,34 @@ case class LociSet(map: LociMap[Long]) {
       (LociSet(mapTake._1), LociSet(mapTake._2))
     }
   }
+
+  /**
+    * Build a collection of HTSJDK Intervals which are closed [start, end], 1-based intervals
+    */
+  def toHtsJDKIntervals: List[Interval] = {
+    map
+      .contigs
+      .iterator
+      .flatMap(
+        contig => {
+          map
+            .onContig(contig)
+            .asMap
+            .iterator
+            // We had 1 to the start to move to 1-based coordinates
+            // Since the `Interval` end is inclusive, we are adding and subtracting 1, no-op
+            .map(pieces => new Interval(contig, pieces._1.start.toInt + 1, pieces._1.end.toInt))
+        }).toList
```

Comment:
minor point: any reason to make this a list? it will use more memory than a Vector